kernel/src/arch/mod.rs

```diff
@@ -10,7 +10,6 @@ pub trait Architecture {
 }
 
 pub trait ArchitectureInterrupts {
-    fn new() -> Self;
```

Comment:
Ok but why?

Reply:
Good point, it should've gone in the other pr